models/issue_comment.go

```diff
@@ -104,7 +106,9 @@ type Comment struct {
 	UpdatedUnix int64     `xorm:"INDEX updated"`
 
 	// Reference issue in commit message
-	CommitSHA string `xorm:"VARCHAR(40)"`
+	CommitSHA    string        `xorm:"VARCHAR(40)"`
+	Link         string        `xorm:"VARCHAR(2048)"`
+	CommitStatus *CommitStatus `xorm:"-"`
```

Comment:
Do we need a migration for this?

Reply:
What's migration? We didn't change anything, the extra fields only for new comment type.

Reply:
You added the Link column. Do we need a migration for syncing this?

Reply:
Currently Link is only for the added comment type.